tensorflow/contrib/distributions/python/ops/beta.py

```diff
@@ -292,3 +293,38 @@ def __init__(self,
           validate_args=validate_args,
           allow_nan_stats=allow_nan_stats,
           name=ns)
+
+
+def _kl_beta_beta(d1, d2, name=None):
+  """Calculate the batched KL divergence KL(d1 || d2) with d1 and d2 Beta.
+
+  Args:
+    d1: instance of a Beta distribution object.
+    d2: instance of a Beta distribution object.
+    name: (optional) Name to use for created operations.
+      default is "kl_beta_beta".
+
+  Returns:
+    Batchwise KL(d1 || d2)
+  """
+  inputs = [d1.a, d1.b, d1.a_b_sum, d2.a_b_sum]
+  with ops.name_scope(
+    name, "kl_beta_beta", inputs):
```

Comment:
don't need this in a new line since the whole with declaration fits within 80 chars.